lib/repl.c

```diff
@@ -76,21 +77,44 @@ void REPL_upload( int flash )
                                            , new_script_len
                                            ) ){
                     printf("flash write failed\n");
-                    Caw_send_luachunk("flash write failed");
+                    Caw_send_luachunk("User script upload failed!");
+                } else {
+                    printf("script saved, len: %i\n", new_script_len);
+                    Caw_send_luachunk("User script updated.");
+                    REPL_print_script_name(NULL);
+                    Lua_crowbegin();
                 }
-                printf("script saved, len: %i\n", new_script_len);
             } else {
-                Caw_send_luachunk("running...");
+                REPL_print_script_name(new_script);
+                Lua_crowbegin();
             }
-            Lua_crowbegin();
         } else {
-            Caw_send_luachunk("evaluation failed");
+            Caw_send_luachunk("User script evaluation failed.");
         }
         free(new_script);
     }
     repl_mode = REPL_normal;
 }
 
+void REPL_clear_script( void )
+{
+    Lua_Reset();
+    Flash_clear_user_script();
+    Caw_send_luachunk("User script cleared.");
+    REPL_print_script_name(NULL);
+    Lua_crowbegin();
+}
+
+void REPL_default_script( void )
+{
+    Lua_Reset();
+    Flash_default_user_script();
+    Caw_send_luachunk("Using default script.");
```

Comment:
Seems like we're using both "default" and "First" for this. Should we just use "First" here for clarity?

Reply:
I'm going to leave it because it is *always* followed by `"Running: First.lua"`. Folks seem to have a hard time with whether 'First' is getting replaced when they upload a script, so calling it the `default` script I think helps with seeing it as 3rd category.